fetch.bs

```diff
@@ -867,7 +867,11 @@ fetch("https://victim.example/naïve-endpoint", {
     </div>
 
    <dt>`<code>range</code>`
-   <dd><p>If <var>value</var> is not a <a>simple range header value</a>, then return false.
+   <dd>
+    <ol>
+     <li><p>Let <var>rangeResult</var> be the result of <a>parse a single range header value</a>.
```

Comment:
We need to pass the parser something to operate on and we usually separate the list items by a newline.
```suggestion
     <li><p>Let <var>rangeResult</var> be the result of <a>parse a single range header value</a>
     given <var>value</var>.

```